tools/mage/proto.go

```diff
@@ -31,7 +31,7 @@ import (
 const (
 	protocOut             = "/out"
 	gogoProtoImage        = "ghcr.io/thethingsindustries/protoc:gen-gogo-1.3.1"
-	jsonProtoImage        = "ghcr.io/thethingsindustries/protoc:3.9.1-gen-go-json-1.1.0"
+	jsonProtoImage        = "ghcr.io/thethingsindustries/protoc:3.9.1-gen-go-json-main"
```

Comment:
NOTE: This will be changed to a different version after https://github.com/TheThingsIndustries/protoc-gen-go-json/pull/5 gets merged.